packages/docs/modules/ROOT/pages/public_deploy.adoc

```diff
@@ -0,0 +1,185 @@
+[[deploy-your-contracts-to-a-public-network]]
+= Deploy your contracts to a public network
+
+This tutorial link:first[builds on the first tutorial] where we created a new OpenZeppelin SDK project, and created a simple `Counter` contract in a local development network. We will now see how to deploy this contract to a public network.  
+
+We will use the Ropsten public test network for this tutorial.  You could also use other public test networks such as Rinkeby, Kovan or Goerli.
+
+[[setup]]
+== Setup
+
+We will use the `my-project` project folder we created earlier.
+
+We will use https://infura.io/[Infura] to interact with public Ethereum nodes (if you are not running your own nodes). 
+Infura Core is free to use. 
+Follow the instructions on the Infura website to sign up and create a new project.  
+
+We will use https://github.com/trufflesuite/truffle/tree/develop/packages/truffle-hdwallet-provider[truffle-hdwallet-provider] to sign transactions for addresses derived from a 12 or 24 word mnemonic.
+Run the following to install as a development dependency:
+
+[source,console]
+----
+$ npm install --save-dev truffle-hdwallet-provider
+----
+
+We will use https://github.com/motdotla/dotenv[dotenv] to store your Infura Project ID and your development mnemonic.
+Run the following to install as a development dependency:
+
+[source,console]
+----
+$ npm install --save-dev dotenv
+----
+
+[[configure-network-js]]
+== Configure network.js
```

Comment:
The file name is `networks.js` (plural), not `network.js`. Make sure to change this across the doc!

Reply:
Oops, sorry about that.  Corrected.

Reply:
Oops, resolved.